app/controllers/simplified-account/settings/email-notifications/email-notifications.controller.js

```diff
@@ -1,19 +1,51 @@
 const { response } = require('../../../../utils/response')
 const humaniseEmailMode = require('../../../../utils/humanise-email-mode')
+const formatSimplifiedAccountPathsFor = require('../../../../utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('../../../../paths')
+const { setEmailCollectionModeByServiceIdAndAccountType } = require('../../../../services/email.service')
+const logger = require('../../../../utils/logger')(__filename)
 
-function get (req, res) {
+function getEmailNotificationsSettingsPage (req, res) {
   const service = req.service
   const account = req.account
 
   const context = {
     emailCollectionMode: humaniseEmailMode(account.email_collection_mode),
     confirmationEmailEnabled: account.email_notifications?.PAYMENT_CONFIRMED?.enabled ?? false,
     refundEmailEnabled: account.email_notifications?.REFUND_ISSUED?.enabled ?? false,
-    isServiceAdmin: req.user.isAdminUserForService(service.externalId)
+    isServiceAdmin: req.user.isAdminUserForService(service.externalId),
+    editEmailCollectionHref: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.emailNotifications.collectionSettings, service.externalId, account.type)
   }
   return response(req, res, 'simplified-account/settings/email-notifications/index', context)
 }
 
+function getEmailCollectionPage (req, res) {
```

Comment:
```suggestion
function getEditEmailCollectionMode (req, res) {
```

Reply:
I prefer `getEmailCollectionPage` as this is what this method is doing - it's simply retrieving the page for the user.

Reply:
I'll change this to `getEditEmailCollectionModePage`